src/rad/resources/schemas/source_catalog-1.0.0.yaml

```
@@ -0,0 +1,20 @@
+%YAML 1.1
+---
+$schema: asdf://stsci.edu/datamodels/roman/schemas/rad_schema-1.0.0
+id: asdf://stsci.edu/datamodels/roman/schemas/source_catalog-1.0.0
+
+title: Source catalog generated by the Source Catalog Step.
+datamodel_name: SourceCatalogModel
+archive_meta: None
+type: object
```

```suggestion
title: Source catalog generated by the Source Catalog Step.

datamodel_name: SourceCatalogModel
archive_meta: None

type: object
```

Makes it a bit easier to read.